hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/metrics/ECBlockGroupsMBean.java

```diff
@@ -58,7 +58,7 @@ public interface ECBlockGroupsMBean {
   long getPendingDeletionECBlocks();
 
   /**
-   * Return total number of erasure coded block groups.
+   * @return total number of erasure coded block groups.
```

Comment:
Nit - this is the only change in this file and all the methods above have the old syntax too. Maybe we should scope this for a different PR to keep the diff clean (and address the entire class)

Reply:
If I don't include this javadoc fails. However even when I do include this a different javadoc fails for essentially same issue.

Reply:
@PrateekSane We should remove this from the patch as it is unrelated to your change. Sorry, I missed this earlier. Everything else looks good and will commit after this change. Thank you!